probe-rs/src/probe.rs

```diff
@@ -842,6 +842,7 @@ pub trait JTAGAccess: DebugProbe {
         &mut self,
         writes: &[JtagWriteCommand],
     ) -> Result<Vec<CommandResult>, BatchExecutionError> {
+        tracing::warn!("Using default `JTAGAccess::write_register_batch` this will hurt performance. Please implement proper batching for this probe.");
```

Comment:
Can we make this a debug statement? Warn would show to every user if they use a probe impl of someone else without this impl :/

Reply:
Done :)